public/pages/QueryGroupDetails/QueryGroupDetails.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CoreStart } from 'opensearch-dashboards/public';
+// @ts-ignore
+import Plotly from 'plotly.js-dist';
+import hash from 'object-hash';
+import { useParams, useHistory, useLocation } from 'react-router-dom';
+import React, {useEffect} from "react";
+import {QUERY_INSIGHTS} from "../TopNQueries/TopNQueries";
+import {
+  EuiButton,
+  EuiCodeBlock,
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiIconTip
+} from '@elastic/eui';
+import QueryGroupAggregateSummary from "./Components/QueryGroupAggregateSummary";
+import QueryGroupSampleQuerySummary from "./Components/QueryGroupSampleQuerySummary";
+
+const QueryGroupDetails = ({ queries, core }: { queries: any; core: CoreStart }) => {
```

Comment:
Why do we need this new component? Is it possible to reuse the QueryDetails page? Cause I see a lot of duplicate logic here with QueryDetails.tsx

Reply:
+1, Unless there is more details that we need to show on the grouping page, but even then we can extract the common parts out into a common component